.circleci/config.yml

```diff
@@ -289,7 +289,7 @@ workflows:
           name: ubuntu-focal-shared-noopenssl
           context: openquantumsafe
           CONTAINER: openquantumsafe/ci-ubuntu-focal-x86_64:latest
-          CMAKE_ARGS: -DCMAKE_C_COMPILER=gcc-7 -DCMAKE_BUILD_TYPE=Release -DOQS_USE_OPENSSL=OFF -DBUILD_SHARED_LIBS=ON
+          CMAKE_ARGS: -DCMAKE_C_COMPILER=gcc-7 -DCMAKE_BUILD_TYPE=Release -DOQS_DIST_BUILD=ON -DOQS_USE_OPENSSL=OFF -DBUILD_SHARED_LIBS=ON
```

Comment:
After today's discussion, isn't "-DCMAKE_BUILD_TYPE=Release -DOQS_DIST_BUILD=ON" a contradiction?

Reply:
It's not so much that Release means "OQS_DIST_BUILD=OFF && OQS_OPT_TARGET=auto" but that Release has no effect and those are the defaults. This is just overriding the default by setting OQS_DIST_BUILD=ON.

There's no reason to ever specify OQS_BUILD_TYPE=Release, so maybe the best thing to do here is remove references to it from the .circleci/config.yml file.